rules/linux/macos_network_service_scanning.yml

```diff
@@ -0,0 +1,25 @@
+title: MacOS Network Service Scanning
+id: 84bae5d4-b518-4ae0-b331-6d4afd34d00f
+status: experimental
+description: Detects enumeration of local or remote network services.
+author: Alejandro Ortuno, oscd.community
+date: 2020/10/21
+references:
+  - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1046/T1046.md
+logsource:
+  category: process_creation
+  product: macos
+detection:
+  selection:
+    ProcessName|endswith:
+      - '/nmap'
+      - '/telnet'
+      - '/nc'
+      - '/netcat'
+  condition: selection
+falsepositives:
+  - Legitimate administration activities
+level: medium
```

Comment:
I believe it should be `low` as it's going to be noisy (ah, yeah. the `level` in the Sigma is not classical severity, it is sort of a mix of severity/confidence/probability of an actual adversary behavior detection; guys don't like when noisy rules have medium and other than `low` severities, even tho the detection rules are destined to detect an adversary that is already inside your infrastructure. most probably that is something that is going to be changed in the near future, but this is how it works at the moment)

Reply:
thank you!